themes/mlibrary_new/custom.php

```diff
@@ -415,27 +412,27 @@ function mlibrary_new_alt_text($item) {
  * Retrieve a thumnail image for a video item type
  *  It is not used in this installation, but it can be used in the future.
  **/
-function mlibrary_new_exhibit_builder_video_attachment($item) {
+function mlibrary_new_exhibit_builder_video_attachment($item,$alt) {
 $remove[] = "'";
 	$elementids_youtube_video = metadata($item, array('Item Type Metadata', 'Video_embeded_code'), array('no_escape'=>true,'all'=>true));
 	$elementvideos_kultura_VCM = metadata($item, array('Item Type Metadata', 'video_embeded_code_VCM'),array('no_escape'=>true, 'all'=>true));
-	$alt_text = mlibrary_new_alt_text($item);
+	//$alt_text = mlibrary_new_alt_text($item);
```

Comment:
This should probably be deleted rather than left as a comment.

Reply:
Agree, I will check if it is the last updated file.